.env.example

```
@@ -12,3 +12,14 @@
 # Prisma
 # https://www.prisma.io/docs/reference/database-reference/connection-urls#env
 DATABASE_URL="postgresql://postgres:password@localhost:5432/org_os"
+
+# WorkOS Authentication
+WORKOS_API_KEY="sk_test_..."
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)